scalafix/rules/src/main/scala/fix/Scalaz2cats_v1_0_0.scala

```diff
@@ -36,7 +36,7 @@ case class MigrateEither(index: SemanticdbIndex) extends SemanticRule(index, "Mi
       case t @ Type.ApplyInfix(lhs, \/(_), rhs) =>
         ctx.replaceTree(t, q"Either[$lhs, $rhs]".syntax)
       case t @ Type.Apply(\/(_), args) =>
-        ctx.replaceTree(t, q"Either[${args(0)}, ${args(1)}]".syntax)
+        ctx.replaceTree(t, q"Either[${args.head}, ${args(1)}]".syntax)
```

Comment:
I just realized we could probably do this with

```scala
case t @ Type.Apply(\/(_), List(rtpe, ltpe)) =>
  ctx.replaceTree(t, q"Either[$rtpe, $ltpe]".syntax)
```

which seems a bit more solid. Would you mind trying?